From 75ec82ede464448f429492345cc871db9d686ca7 Mon Sep 17 00:00:00 2001
From: Pawit Pornkitprasan <p.pawit@gmail.com>
Date: Sun, 11 Oct 2015 21:11:25 +0700
Subject: [IGNORE THIS PATCH 11/11] HACK: temporarily change SELinux to permissive

---
 init/init.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/init.cpp b/init/init.cpp
index 80d21be..61e465c 100644
--- a/init/init.cpp
+++ b/init/init.cpp
@@ -894,7 +894,7 @@ static void selinux_init_all_handles(void)
 enum selinux_enforcing_status { SELINUX_DISABLED, SELINUX_PERMISSIVE, SELINUX_ENFORCING };
 
 static selinux_enforcing_status selinux_status_from_cmdline() {
-    selinux_enforcing_status status = SELINUX_ENFORCING;
+    selinux_enforcing_status status = SELINUX_PERMISSIVE;
 
     std::function<void(char*,bool)> fn = [&](char* name, bool in_qemu) {
         char *value = strchr(name, '=');
-- 
2.3.8 (Apple Git-58)
