From 02947b50ae61679a322d188b14aac7b2ec34e820 Mon Sep 17 00:00:00 2001
From: k2wl <k2wl@github.com>
Date: Thu, 4 Sep 2014 14:27:19 +0530
Subject: [PATCH] updated patch

Change-Id: I67a3f898841170b84077b569e6d356a4afcf0393
Signed-off-by: k2wl <k2wl@github.com>
---
 qcom-common.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/qcom-common.mk b/qcom-common.mk
index 12f426f..5bf5f9c 100644
--- a/qcom-common.mk
+++ b/qcom-common.mk
@@ -28,7 +28,6 @@ PRODUCT_PROPERTY_OVERRIDES += \
 
 PRODUCT_PROPERTY_OVERRIDES += \
     ro.telephony.call_ring.multiple=0 \
-    ro.telephony.ril_class=SamsungQualcommRIL
 
 # Charger
 PRODUCT_PACKAGES += \
-- 
1.9.1

