From 36290569eea10ee3e9889b9f179986e7c305a992 Mon Sep 17 00:00:00 2001
From: k2wlxda <k2wlxda@gmail.com>
Date: Thu, 9 Jul 2015 12:44:51 +0530
Subject: [PATCH] Disable block based zip

Change-Id: Idd7b4955799884684d9c1f6f6da3b29d9016887e
Signed-off-by: k2wlxda <k2wlxda@gmail.com>
---
 core/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/Makefile b/core/Makefile
index 7b25927..27d27e0 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1689,7 +1689,6 @@ $(INTERNAL_OTA_PACKAGE_TARGET): $(BUILT_TARGET_FILES_PACKAGE) $(DISTTOOLS)
 	@echo -e ${CL_YLW}"Package OTA:"${CL_RST}" $@"
 	$(hide) MKBOOTIMG=$(MKBOOTIMG) \
 	   $(OTA_FROM_TARGET_SCRIPT) -v \
-	   --block \
 	   -p $(HOST_OUT) \
 	   -k $(KEY_CERT_PAIR) \
 	   --backup=$(backuptool) \
-- 
2.4.0

