From 6c7af2ddaa10eba6b10a729d3b095a6131c00638 Mon Sep 17 00:00:00 2001
From: k2wl <kar2wlan@gmail.com>
Date: Fri, 18 Dec 2015 08:15:36 +0530
Subject: [IGNORE THIS PATCH] Remove SerranoRIL Prop

Signed-off-by: k2wl <kar2wlan@gmail.com>
---
 system_prop.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/system_prop.mk b/system_prop.mk
index 0b0ba50..2cb1f3a 100644
--- a/system_prop.mk
+++ b/system_prop.mk
@@ -8,7 +8,6 @@ PRODUCT_PROPERTY_OVERRIDES += \
 
 # RIL
 PRODUCT_PROPERTY_OVERRIDES += \
-    ro.telephony.ril_class=SerranoRIL \
     ro.ril.telephony.mqanelements=6 \
     persist.radio.add_power_save=1
 
-- 
2.5.0

